doc/user/install.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Install"
+description: "Get started with Materialize"
+menu: "main"
+weight: 2
+---
+
+You can access Materialize through the `materialized` binary, which you can install on macOS and Linux.
```

Comment:
I think we should mention that we have unofficial support for other operating systems, like BSD, if you compile from source.